frontend/src/clients/PipelineToolClient.ts

```diff
@@ -25,8 +25,8 @@ export class PipelineToolClient extends HttpClient {
       if (code === 400) {
         throw new BadRequestException(params.type, 'Bad request')
       }
-      if (code === 404) {
-        throw new NotFoundException(params.type, 'Page not found')
+      if (code === 401) {
+        throw new NotFoundException(params.type, 'Token is incorrect')
```

Comment:
`Incorrect` means the value is not valid
`401` means not `Unauthorized` & the token's format or other things are correct
`NotFoundException` === 401, are u kidding?

Reply:
Double check with that, 404 => not found, 401 => Unauthorized, fix it